src/integrationTest/java/org/openstreetmap/atlas/generator/AtlasGeneratorIntegrationTest.java

```diff
@@ -87,10 +94,19 @@ public void testAtlasGeneration()
                     Iterables.size(resourceForName(resourceFileSystem,
                             LINE_DELIMITED_GEOJSON_OUTPUT + "/9/DMA_9-168-234.ldgeojson.gz")
                                     .lines()));
-            Assert.assertEquals(402,
+            Assert.assertEquals(336,
```

Comment:
any reason for this change?

Reply:
yes. the method below `resourceForName` had the shard 233 path hardcoded by mistake. Now it really loads the one asked for, and as a result, the 234 numbers are different....

Reply:
that makes sense!